spec/spec-helper.coffee

```diff
@@ -58,7 +58,7 @@ keydown = (key, {element, ctrl, shift, alt, meta, raw}={}) ->
   key = "U+#{key.charCodeAt(0).toString(16)}" unless key is 'escape' or raw?
   element ||= document.activeElement
   eventArgs = [
-    true, # bubbles
+    false, # bubbles
```

Comment:
Please, note that I had to disable bubbling here because, for some reason, events were being fired twice. I guess this has something to do with attaching the element to the DOM but I haven't quite understood exactly what was causing it.

@maxbrunsfeld: I guess I can have a second look if you think it's needed, but this change didn't seem that bad either.